zowe-versions.yaml

```diff
@@ -88,17 +88,17 @@ packages:
     next: true
     zowe-v1-lts: 4.0.11
     zowe-v2-lts: 5.0.6
-    zowe-v3-lts: 6.0.0
+    zowe-v3-lts: 6.0.0-next.202403011419
   cics-for-zowe-cli:
     next: true
     zowe-v1-lts: 4.0.11
     zowe-v2-lts: 5.0.6
-    zowe-v3-lts: 6.0.0
+    zowe-v3-lts: 6.0.0-next.202403011419
   db2-for-zowe-cli:
     next: true
     zowe-v1-lts: 4.1.13
     zowe-v2-lts: 5.0.9
-    zowe-v3-lts: 6.0.0
+    zowe-v3-lts: 6.0.0-next.202408142011
```

Comment:
The prerelease for DB2 plug-in exists, but was never tagged as `next` on NPM 🤔 
![image](https://github.com/user-attachments/assets/74340ecb-29c0-4fd4-b837-21fd65530e81)